### PR TITLE
add darkhorse call for candidate genomes

### PR DESCRIPTION
**Pull request by ekopylova**

### CODE REVIEW

File: benchmark/parse_output.py [hunk 1]
```diff
@@ -101,21 +100,34 @@ def parse_darkhorse(input_f, low_lpi=0.0, high_lpi=0.6, return_genomes=False):
...
     """
-    return None
+    best_hit_ids = set()
+    hgts = []
+    # skip header
+    next(input_f)
+    for l in input_f:
+        l = l.strip('\r\n').split('\t')
+        best_hit_ids.add(l[3])
+        if (float(l[5]) > low_lpi) and (float(l[5]) < high_lpi):
```
Comment (qiyunzhu) [inline benchmark/parse_output.py]: How about `if low_lpi < float(l[5]) < high_lpi:`?

File: benchmark/parse_output.py [hunk 2]
```diff
@@ -101,21 +100,34 @@ def parse_darkhorse(input_f, low_lpi=0.0, high_lpi=0.6, return_genomes=False):
...
     Returns
     -------
-    number_of_hgts: string
-        number of HGTs reported by a tool, or NaN if an entry was not found
-
+    hgts: string
+        one putative HGT-derived gene per line
+        columns: query_id, besthit_id, tax_id, species, lineage, pct_id,
+        pct_coverage, norm_LPI
     Notes
```
Comment (qiyunzhu) [inline benchmark/parse_output.py:114]: Need empty line between sections.

**Review by qiyunzhu — COMMENTED** [review]
Looks good!

**Comment by RNAer** [comment]
👍  don't have any other comments

**Pull request merged**